astropy/io/registry.py

```diff
@@ -342,7 +352,21 @@ def _get_format_table_str(data_class, readwrite):
 
 
 def get_reader(data_format, data_class):
-    # Get all the readers that work for `data_format`
+    """Get reader for ``data_format``.
+
+    Parameters
+    ----------
+    data_format : str
+        The data type identifier. This is the string that is used to
+        specify the data type when reading/writing.
```

Comment:
Should probably be `data format` everywhere, not `data type`, since the latter could be confused with the data class.